tests/suite/system_suite_test.go

```diff
@@ -303,5 +303,6 @@ func isNFR(labelFilter string) bool {
 		strings.Contains(labelFilter, "longevity") ||
 		strings.Contains(labelFilter, "performance") ||
 		strings.Contains(labelFilter, "upgrade") ||
-		strings.Contains(labelFilter, "scale")
+		strings.Contains(labelFilter, "scale") ||
+		strings.Contains(labelFilter, "ip-family")
```

Comment:
This isn't an NFR test, it's a functional test.

Does this test require GKE? There's no way to run it in kind?